docs/_plugins/p2export.md

```diff
@@ -0,0 +1,216 @@
+
+    update          =   https://bndtools.jfrog.io/bndtools/update-latest
+    update.label    =   Bndtools Update Site
+
+Each feature can participate in a number of categories. The category definitions look as follows:
+
+    -categories \
+        bndtools; \
+            label = Bndtools; \
+            description = Bndtools, the incredible IDE for OSGi
+
+This can be repeated by adding more clauses. A feature's bndrun file must set the Bundle-Category that is then matched in this list fo the label and description.
+
+And last but not least, general header files.
+
+    Bundle-Version  7.0.0
+    Bundle-License:  \
+        ASL-2.0;\
+            description="This program and the accompanying materials are made available under the terms of the Apache License, Version 2.0";\
+            link="https://opensource.org/licenses/Apache-2.0", \
+        EPL-2.0;\
+            description="This program and the accompanying materials are made available under the terms of the Eclipse License, Version 2.0";\
+            link="https://opensource.org/licenses/EPL-2.0"
+            
+    
+    Bundle-DocURL:          https://bnd.bndtools.org/
+    Bundle-Vendor           bnd
+    Bundle-Copyright:       Copyright bndtools
+
+
+These headers are inherited in the features and can be overridden.
+
+## Feature Definitions
+
+Please note that the features you specify must be valid bndrun/feature files. The bndrun files inherit from the Workspace. It is therefore possible to set shared values in the workspace and use them in the bndrun files. For example, you can set the `Bundle-Version` in the workspace and use it in all the feature bndrun files.
+
+Although the features are not bundles, the exporter treats them as such. This means that most of the OSGi headers used for bundles can also be used for features. For example, you can set the `Bundle-License` header in the manifest file of a feature to specify licensing information for that feature.
+
+The following headers can be used in the feature files.
+
+* `Bundle-Name` – Human readable name of the feature
+* `Bundle-SymbolicName` – Used for the feature id. This is default the same as the name of the feature/bndrun file.
+* `Bundle-Description` – A human readable description of the feature
+* `Bundle-DocUrl` – A URL to the documentation of the feature
+* `Bundle-Copyright` – The copyright
+* `Bundle-License` – The licenses
+* `Bundle-Category` – The category of the feature
+* `Bundle-Vendor` – The vendor will be used as the provider of the feature
+
+The format of these headers is exactly as outlined by their OSGi specification.
+
+Features can require bundles and other features. 
+
+Bundles will come from the `-runbundles` instruction. This instruction can be managed with the resolver but it can also be set manually. Standard rules apply. The workspace is consulted to find the bundles and their versions. The bundles will be the highest available version in the repository.
```

Comment:
As a side note: 
There are two kinds of P2 repos. One is a complete repository, that contains all transient dependencies. The second one, like the bnd P2 Repo just contains a set of bundles. The dependencies are resolved in all the other P2 repositories in a target platform.
I mentioned, that I also create an unfinished Exporter:
https://gitlab.com/gecko.io/geckoannotations/-/tree/develop/org.gecko.artifact.p2.exporter?ref_type=heads
I've also chosen the approach of taking -runbundles but only for the complete repository. For the one that should only contain my bundles and no transient dependencies I used the -runrequire instruction:
https://gitlab.com/gecko.io/geckoannotations/-/blob/develop/org.gecko.artifact.p2.exporter/src/org/gecko/artifact/p2/exporter/P2Exporter.java?ref_type=heads
I experienced, that the P2 resolver tries to resolve all dependencies against the target platform repos.

Reply:
Using `-runbundles` does not preclude using `-runrequires`. The resolve operation sets the `-runbundles`?

It is a pity that I did not know this code ... I could not found the org.gecko.artifact.p2.P2RepositoryCreator though and most of the work seems to happen there?